routes/api/v1.js

```diff
@@ -223,9 +222,9 @@ api.declare({
     "Modify client `name`, `expires`, `scopes` and",
     "`description`.",
     "",
-    "**Required scopes**, in addition the scopes listed",
-    "above, the caller must also posses the all the scopes that is given to",
-    "the client that is updated."
+    "**Required scopes**: in addition the scopes listed",
+    "above, the `scopes` property must be satisfied by the caller's",
+    "scopes.  The client's existing scopes are not considered."
```

Comment:
This is nit, but if we do have colon, I believe one should follow with a capital "In", not sure though... Either way, I was happy with comma and remains happy with colon :)

Note, `satisfied by the caller` is likely sufficient to imply satisfied by the caller's scopes. But again, it's nit picking...